### PR TITLE
Enable cache busting for i18n json files

### PR DESCRIPTION
**Pull request by thmsltmnn**

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -39,8 +39,8 @@ export * from './src/component';
 export * from './src/language';
 export * from './src/interceptor';
 
-export function translatePartialLoader(http: Http) {
-    return new TranslateHttpLoader(http, 'i18n/', '.json');
+export function translatePartialLoader(http: Http, configService: JhiConfigService) {
+    return new TranslateHttpLoader(http, 'i18n/', '.json?build=' + configService.getConfig().i18nCacheBuster);
```
Comment (deepu105) [inline index.ts]: use string template instead of concatenation here

**Comment by thmsltmnn** [comment]
Changed concatenation to string template.

File: src/config.ts [hunk 2]
```diff
@@ -27,5 +27,6 @@ export class JhiModuleConfig {
     i18nEnabled? = false;
     defaultI18nLang? = 'en';
     noi18nMessage? = 'translation-not-found';
+    i18nCacheBuster? = '';
```
Comment (ctamisier) [inline src/config.ts]: Should it be called something like `buildTimestamp` to eventually use it for something else than cache buster ?
(and hitting the cache is avoided only the first request after deployment)

Comment (thmsltmnn) [reply src/config.ts]: Fine with me. We could call it `buildTimestamp` for the config as well as the URL parameter.

**Pull request merged**